python_modules/dagster/dagster/_core/execution/plan/instance_concurrency_context.py

```diff
@@ -122,3 +125,14 @@ def free_step(self, step_key) -> None:
 
         self._instance.event_log_storage.free_concurrency_slot_for_step(self._run_id, step_key)
         self._claims.remove(step_key)
+
+
+def _calculate_timeout_interval(sleep_interval: Optional[float], pending_claim_count: int) -> float:
+    if sleep_interval is not None:
+        return sleep_interval
+
+    if pending_claim_count > 100:
+        return MAX_CONCURRENCY_CLAIM_BLOCKED_INTERVAL
```

Comment:
this gets up past MAX_CONCURRENCY_CLAIM_BLOCKED_INTERVAL way before 100 claim counts no? what's the significance of 100 here?

Reply:
yeah, maybe makes more sense to set it to the actual value.  I was just thinking that we might want to change the initial value and would not want to be capped by it.  But also not perform unnecessary compute here